packages/database/src/core/RepoInfo.ts

```diff
@@ -133,3 +91,51 @@ export class RepoInfo {
     return `${protocol}${this.host}/${query}`;
   }
 }
+
+function needsQueryParam(repoInfo: RepoInfo): boolean {
```

Comment:
Nit: I would also prefix this with "repoInfo" for consistency.

Reply:
Done.